### PR TITLE
[ISSUE 8872][docs] fix the json format of BrokerClientAuthParam

### PR DESCRIPTION
**Pull request by sijia-w**
#8872 

Modification:
Update the format in the versioned docs and the master branch.

### CODE REVIEW

**Comment by Huanli-Meng** [comment]
It seems Jeniffer has made a similar doc update. Please hep align with Jeniffer to see if there is any overlap between your work.
@Jennifer88huang , please help double check. Thanks.

**Review by Jennifer88huang — APPROVED** [review]
If you've confirmed with eng that we should change all `brokerClientAuthenticationParameters` value as JSON format, that's OK.

Anyway, do not merge it until there is at least one eng approve it.

**Review by sijie — APPROVED** [review]

**Pull request merged**
